packages/web-components/fast-foundation/src/design-system/design-system.ts

```diff
@@ -90,16 +108,10 @@ export interface DesignSystemRegistrationContext {
 
     /**
      * Used to attempt to define a custom element.
-     * @param name - The name of the element to define.
-     * @param type - The type of the constructor to use to define the element.
-     * @param callback - A callback to invoke if definition will happen.
+     * @param ctx - The definition context.
```

Comment:
I think we need to fix this param description since this is technically not the definition context.

Reply:
I had a bit of a hard time finding a proper description for this one. I could use some suggestions, otherwise I'll come up with something on monday 👍

Reply:
Maybe just using `Element Definition`? @EisenbergEffect any idea?

Reply:
I highly inspired from @vaudy 's suggestion, but feel free to suggest anything else 👍